### ISSUE TITLE
[FRONT] Pensar nas telas

### ISSUE THREAD

**Issue by ernegreiros** [ernegreiros/BatalhaNaval#4]
Precisamos pensar na tela inicial do sistema 
**Ideia**
Tela 1 -> Home com um código e um campo para adicionar o codigo de outra pessoa 

Tela 2 -> Tela do jogo, uma matriz ocupando a maior parte dela, com o campo do batalha naval 

Só precisariamos dessas duas telas

Telas iniciadas:
![image](https://user-images.githubusercontent.com/43824634/92857182-b98a1880-f3ca-11ea-8274-78cc9bebec29.png)

![image](https://user-images.githubusercontent.com/43824634/92857264-cf97d900-f3ca-11ea-98dc-7c05563174bc.png)

**Comment by luizAGT** [comment]
Ed...não sei se vou dizer besteira: ao invés de usar código para entrar, não ficaria melhor pedir o nome do jogador para iniciar o jogo?

**Comment by ernegreiros** [comment]
Sim, é uma ideia valida! O pensamento inicial é seguir o modelo adotado nesses jogos:
- Among Us
![image](https://user-images.githubusercontent.com/43824634/95915606-c4dfb380-0d7d-11eb-9bc5-d4436edebc56.png)
![image](https://user-images.githubusercontent.com/43824634/95915579-b8f3f180-0d7d-11eb-8bc0-b47b90fc7fb8.png)

- Drawful 2
![image](https://user-images.githubusercontent.com/43824634/95915633-d032df00-0d7d-11eb-9c2f-22b31a41b635.png)
![image](https://user-images.githubusercontent.com/43824634/95915654-d759ed00-0d7d-11eb-93f8-e7c7f78d515a.png)

Onde voce cria uma "sala" e as pessoas entram no seu jogo, mas a forma como vamos implementar isso pro usuario, pode ser mudada facilmente, podemos usar o login do jogador, uma gamertag, um codigo aleatorio... o que for melhor
